run-bahmni.sh

```diff
@@ -86,25 +86,25 @@ function showOpenMRSlogs {
 
 function startMart {
     echo "Starting services with profile 'bahmni-mart'..."
-    docker compose --profile bahmni-mart up -d
+    docker compose --env-file "$file" --profile bahmni-mart up -d
 }
 
 function pullLatestImages {
-    echo "Pulling all latest images..."
-    docker compose pull
+    echo "Pulling all the images specified in the $file file..."
+    docker compose --env-file "$file" pull
 }
 
 function showStatus {
     echo "Listing status of running Services with command: 'docker compose ps'"
     # Using all profiles, so that we can status of all services
-    docker compose --profile bahmni-lite --profile bahmni-standard --profile bahmni-mart ps
+    docker compose --env-file "$file" --profile bahmni-lite --profile bahmni-standard --profile bahmni-mart ps
 
 }
 
 
 # Function to prompt the user for a "Yes" or "No" answer
 confirm() {
-    read -p "$1 [y/n]: " response
+    read -p "$file [y/n]: " response
```

Comment:
Why is this changed to $file?

Reply:
This was done by mistake, will revert this.